src/tests/suite/daos_obj_ec.c

```diff
@@ -627,15 +695,27 @@ ec_full_partial_update_agg(void **state)
 		memset(buffer, 'a' + i, partial_update_size);
 		memcpy(verify_buffer, buffer, partial_update_size);
 
-		insert_recxs("d_key", "a_key", 1, DAOS_TX_NONE, &recx, 1,
+		insert_recxs("d_key1", "a_key", 1, DAOS_TX_NONE, &recx, 1,
+			     buffer, partial_update_size, &req);
+		insert_recxs("d_key2", "a_key", 1, DAOS_TX_NONE, &recx, 1,
 			     buffer, partial_update_size, &req);
 	}
 
-	trigger_and_wait_ec_aggreation(arg, &oid, 1, "d_key", "a_key", 0,
-				       full_update_size, DAOS_FORCE_EC_AGG);
-
-	ec_verify_parity_data(&req, "d_key", "a_key", (daos_size_t)0,
-			      full_update_size, verify_data, DAOS_TX_NONE, true);
+	trigger_and_wait_ec_aggreation_2dkeys(arg, &oid, 1, "d_key1", "d_key2", "a_key", 0,
```

Comment:
So to be clear, because this was discussed in chat, there is no way to trigger this with one failure and 1 parity target?   Multiple dkeys alone is not a sufficient condition?

Reply:
multiple dkey is not a sufficient cond, need multiple dkeys share a parity shard and with different parity shard, select the shared shard as agg leader.
"no way to trigger this with one failure" possibly trigger with one failure, but with less possibility.